### ISSUE TITLE
Proposal: Cut 0.3.0 Release.

### ISSUE THREAD

**Issue by markmandel** [googleforgames/quilkin#444]
It's been a few month's since we cut a release, and we said we would review at each community meeting. In lieu of that, I figured an issue would suffice 😄 and also figured it would be nice to cut a release before end of year.

I think we have enough new features - some nice API changes (dynamic metadata), the new Firewall filter, some security fixes.

Things that are flowing through now that I think we should wait on before release:

* https://github.com/googleforgames/quilkin/issues/443 - just because 404's on docs aren't good
* https://github.com/googleforgames/quilkin/issues/292 - This is in PR right now (#441), and I have the grafana chart changes in the wings to come shortly after.
* https://github.com/googleforgames/quilkin/issues/367 - I don't think we need to wait for the actual CI test integration, but I have a bunch of fixes for 404's in docs coming likely today.
* https://github.com/googleforgames/quilkin/pull/437 - This is almost ready to go, so we should wait on this 👍🏻 .

I don't _think_ there is anything else, WDYT?

If that sounds good, I'll start a milestone and start to line things up in it.

**Comment by markmandel** [comment]
Update: Forgot the xDS updates aren't merged yet, but we should wait for that too.

**Comment by markmandel** [comment]
Talked about in community call - we will cut a release.

**Comment by markmandel** [comment]
Only blocking PR for release is:

* https://github.com/googleforgames/quilkin/pull/472

Nice to have:

* https://github.com/googleforgames/quilkin/pull/484
